### DISCUSSION TITLE
Remove default options in launcher menu

### DISCUSSION THREAD

**Discussion by pwsandoval** [wezterm/wezterm#3175] [General]
Is posible remove the default options in launcher menu, i.e. with this

```lua
launch_menu = {
      {
         label = "PowerShell Core",
         args = { "pwsh" },
      },
```

I get my shell in first place, but also another unnecessary long list:

![Screenshot_1](https://user-images.githubusercontent.com/15174704/221755831-f0e7b598-c573-446e-8759-b815284d99cc.png)

**Comment by wez** [comment]
You can bind your key to https://wezfurlong.org/wezterm/config/lua/keyassignment/ShowLauncherArgs.html instead

**Reply by pwsandoval** [reply]
Thanks man, I spent a while looking for this and couldn't find it in the documentation.

```lua
{
      key = "Space",
      mods = "CTRL",
      action = wezterm.action.ShowLauncherArgs { flags = 'LAUNCH_MENU_ITEMS' },
   },
```

**Reply by ghost** [reply]
For what it's worth I have this approach building on this:
```
config.keys = {
	{
	  key = 'Space',
	  mods = 'ALT',
	  action = act.ShowLauncherArgs { flags = 'FUZZY|TABS' },
	},
	{
		key = "Space",
		mods = "CTRL",
		action = wezterm.action.ShowLauncherArgs { flags = 'LAUNCH_MENU_ITEMS|FUZZY' },
	 },
  }
```

This allows me to search open tabs or launch menu items
